assets/js/googlesitekit/modules/datastore/modules.js

```diff
@@ -231,8 +231,9 @@ const baseActions = {
 				active
 			);
 			if ( response?.success === true ) {
-				// Fetch (or re-fetch) all modules, with their updated status.
-				yield fetchGetModulesStore.actions.fetchGetModules();
+				// Fetch (or re-fetch) all modules, with their updated status is temporary disabled until Site Kit no longer relies on page reloads between module activation changes.
```

Comment:
The note about this being temporary disabled should go in a separate sentence, which according to the IB we should stick a "TODO" onto as well 🙂 

```suggestion
				// Fetch (or re-fetch) all modules, with their updated status.
				// TODO: This is temporary disabled until Site Kit no longer relies
				// on page reloads between module activation changes.
```